### ISSUE TITLE
I can't manually add a met mast to the AssetData

### ISSUE THREAD

**Issue by charlie9578** [NREL/OpenOA#150]
I think it would be useful to be able to manually add AssetData. For example if I'm bringing in turbine data from one place, but mast data from another, and it would be nice to be able to do this in the notebook. For instance:

 ```
import pandas as pd

from operational_analysis.toolkits.pandas_plotting import plot_windfarm
from operational_analysis.types import PlantData

from project_ENGIE import Project_Engie

# Load plant object
project = Project_Engie('../examples/data/la_haute_borne')

# Prepare data
project.prepare()

# Adding a pseudo met mast (tower) as an example *fails - as no setter*
project.asset.df = project.asset.df.append(pd.DataFrame({'id':['Mast01'],'type':['tower'],'longitude':[5.5],'latitude':[48.5]}))

```

**Comment by ejsimley** [comment]
This is a good suggestion. One option could be to add a setter for df, or another option could be to make an `add_assets` method that requires at a minimum the id, type, and coordinates (there is already a `remove_assets` method). Not the right way to do this, but a workaround for now to help you test your code could be to directly assign to project.asset._asset

**Comment by jordanperr** [comment]
Hi Charlie, as discussed, I think the best approach is either (1) Write out the asset data to a format that Pandas can read (like CSV) and then use project.asset.load(). Or (2) you can reach in and set asset._df directly. I agree that an API change is probably warranted to allow direct manipulation of the dataframes. Adding a setter method is a great suggestion.

**Comment by jordanperr** [comment]
This should be closed with PR #162 - please re-open this ticket if the issue is not solved. Thank you!
